x-pack/plugins/security_solution/scripts/run_cypress/parallel_serverless.ts

```diff
@@ -571,6 +571,7 @@ ${JSON.stringify(cypressConfigFile, null, 2)}
               KIBANA_PASSWORD: credentials.password,
 
               CLOUD_SERVERLESS: true,
+              IS_SERVERLESS: true,
```

Comment:
Cant we use only one of these two flags in lines 573 and 574? If this is not a big rework.

Reply:
These flags represent two different things.
- `IS_SERVERLESS` is used for both FTR and cloud environments. 
- `CLOUD_SERVERLESS` is used for just deployed projects on the cloud, this cannot be used for the FTR serverless environment.